windows/src/global/delphi/packages/Keyman.System.PackageInfoRefreshKeyboards.pas

```diff
@@ -294,6 +294,9 @@ class function TPackageInfoRefreshKeyboards.FillKeyboardDetails(f: TPackageConte
 var
   ki: TKeyboardInfo;
 begin
+  if not FileExists(f.FileName) then
```

Comment:
This addresses a crash in Keyman Developer when loading a package source file where the compiled keyboards are not currently available.